components/Icons.es6.js

```diff
@@ -1941,6 +1941,92 @@ function ArrowCircleUp ({ className, width='16', height='16', color = '#766bff',
   )
 }
 
+function ArrowUp ({ className, width='16', height='16', color = '#766bff', viewBox='0 0 16 16' }) {
+  return (
+    <svg width={width} height={height} viewBox={viewBox} fill="none" xmlns="http://www.w3.org/2000/svg" className={className}>
+      <g clipPath="url(#clip0_130_15802)">
```

Comment:
This should be stripped out. The icon should be pure vector.
In figma, use "Flatten" option on object group to convert it to vector lines.